charts/trino/README.md

```diff
@@ -63,6 +63,8 @@ The following table lists the configurable parameters of the Trino chart and the
 | `coordinator.resources` |  | `{}` |
 | `coordinator.livenessProbe` |  | `{}` |
 | `coordinator.readinessProbe` |  | `{}` |
+| `coordinator.lifecycle` |  | `{}` |
+| `coordinator.terminationGracePeriodSeconds` |  | `""` |
```

Comment:
The actual default is 30, we should put it in `values.yaml`.

Reply:
Sure, I'll include it in `values.yaml`.

Reply:
Done!